diff --git a/br2_ext/src/nrc/nrc-hif-cspi.c b/br2_ext/src/nrc/nrc-hif-cspi.c
index 88afce8..781c115 100644
--- a/br2_ext/src/nrc/nrc-hif-cspi.c
+++ b/br2_ext/src/nrc/nrc-hif-cspi.c
@@ -533,7 +533,7 @@ static int _c_spi_read_regs(struct spi_device *spi,
 
 	arr_len = (size > 1) ? ARRAY_SIZE(xfer) : 2;
 	status = spi_sync_transfer(spi, xfer, arr_len);
-	printk(KERN_ERR "WZab STF: %x\n buffer:%x,%x,%x,%x,%x,%x,%x,%x\n",(int) status, (int)rx[0],(int)rx[1],(int)rx[2],(int)rx[3],(int)rx[4],(int)rx[5],(int)rx[6],(int)rx[7]);
+	//printk(KERN_ERR "WZab STF: %x\n buffer:%x,%x,%x,%x,%x,%x,%x,%x\n",(int) status, (int)rx[0],(int)rx[1],(int)rx[2],(int)rx[3],(int)rx[4],(int)rx[5],(int)rx[6],(int)rx[7]);
 	if (status < 0 || WARN_ON(rx[7] != C_SPI_ACK))
 		return -EIO;
 
